src/user/userRoutes.js

```diff
@@ -0,0 +1,7 @@
+const { Router } = require("express");
+const { addUser } = require("./userController");
+const userRouter = Router();
+
+userRouter.post("/user", addUser);
```

Comment:
endpoints might need to change, can leave as is for now, but might need to change it to "/users" not "/user" if we want to follow RESTful principles